feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/widget/SessionsItemDecoration.kt

```diff
@@ -38,18 +38,17 @@ class SessionsItemDecoration(
 
     override fun onDrawOver(c: Canvas, parent: RecyclerView, state: RecyclerView.State) {
         super.onDrawOver(c, parent, state)
-        var lastStartTimeText: StartTimeText? = null
-        parent.forEach { view ->
+        parent.forEachIndexed { index, view ->
             val position = parent.getChildAdapterPosition(view)
             if (position == RecyclerView.NO_POSITION) return
 
             val sessionItem = adapter.getItem(position) as SessionItem
             val startTimeText = calcTimeText(position, view)
 
-            lastStartTimeText?.let {
-                if (sessionItem.startSessionTime() == it.value) return@forEach
+            if (index > 0) {
```

Comment:
First, I tried
```
val layoutManager = parent.layoutManager as LinearLayoutManager
val topVisibleItemPosition = layoutManager.findFirstVisibleItemPosition()
if (position > topVisibleItemPosition) {
    val lastSessionItem = adapter.getItem(position - 1) as SessionItem
    if (sessionItem.startSessionTime() == lastSessionItem.startSessionTime()) return@forEach
}
```
but, first element of forEach and element at topVisibleItemPosition are not same, so flickering occured when scrolling. So I use `forEachIndexed` and check top element using index.